encord/orm/dataset.py

```diff
@@ -765,6 +765,15 @@ class DatasetData(base_orm.BaseORM):
     )
 
 
+class DatasetDataLongPolling(base_orm.BaseORM):
+    DB_FIELDS = OrderedDict(
+        [
+            ("is_done", bool),
+            ("response", dict),
```

Comment:
Can the response not be typed? Isn't it a `AddPrivateDataResponse` ?

Reply:
This is the "intermediate", "incomplete" response. I think this is fine?

Reply:
Obvs the "this should be a dataclass and not a base-orm" point still stands.

Reply:
It's like Alexey said not final but intermediate response, used internally not returned to user.
Returned response has AddPrivateDataResponse type